Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM ubuntu:latest
 
-COPY ./target/debug/server /usr/local/bin/server
+COPY ./target/release/server /usr/local/bin/server
```

Comment:
Now this errors when doing docker-compose in the source distribution (which has server in ./target/debug/server). Need to have support for both source and binary distros with a conditional